### PR TITLE
ICMSLST-560 Add edit goods section of derogation from sanctions ban.

### PR DESCRIPTION
**Pull request by trbateman**

### CODE REVIEW

File: web/templates/web/domains/case/import/derogations/edit_derogations.html [hunk 1]
```diff
@@ -20,7 +20,19 @@ <h4>Sanctions Derogation</h4>
...
+
+                        Please upload your contract in the Supporting Documents section below.
+
+    </div>
+
+    <h4>Goods</h4>
+    {{ fields.field(form.commodity_code) }}
+    {{ fields.field(form.goods_description) }}
+    {{ fields.field(form.quantity) }}
+    {{ fields.field(form.value) }}
```
Comment (kevinetienne) [inline web/templates/web/domains/case/import/derogations/edit_derogations.html:33]: It looks like you need to include the unit in the form for the user?

File: web/domains/case/_import/derogations/forms.py [hunk 2]
```diff
@@ -40,6 +41,32 @@ class DerogationsForm(forms.ModelForm):
         widget=forms.Textarea(attrs={"cols": 50, "rows": 3}),
     )
 
+    # TODO restrict commodities to group when it becomes known
```
Comment (kevinetienne) [inline web/domains/case/_import/derogations/forms.py:44]: Another solution could be to hardcode the commodities in a choicefield instead as done in https://github.com/uktrade/icms/blob/6513be8bb47c417631b32737d4946f7838a5785c/web/domains/case/_import/wood/forms.py#L45-L53

**Review by kevinetienne — COMMENTED** [review]
I think you need to add the unit to help the user filling the form.

File: web/templates/web/domains/case/import/derogations/edit_derogations.html (hunk 1 above)
Comment (trbateman) [reply web/templates/web/domains/case/import/derogations/edit_derogations.html:33]: Discuss to make these changes on the next ticket

**Review by kevinetienne — APPROVED** [review]
`commodity_code` and `units` for goods would be addressed in a separate ticket.

**Pull request merged**